### PR TITLE
Made changes to Atomic App instructions, run command in response to

### PR DESCRIPTION
**Pull request by jberkus**
issue #274 

Tomas, please try the updated instructions and docker-run command to see if it fixes the run for you.

@TomasTomecek

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -33,6 +34,7 @@ Once you've done the setup, you should be able to do this to run the container
 each time and test the atomic site by running from the repository root:
 
 ```
+    cd /path/to/atomic-site/
     sudo atomicpp run Nulecule/
```
Comment (TomasTomecek) [inline BUILD.md]: typo, should be `atomicapp`

File: BUILD.md [hunk 2]
```diff
@@ -6,22 +6,23 @@ instructions for building it.
...
 
-1. authorize directory sharing in SELinux (may not be required in a VM):
+1. run genanswers:
 
-```
-    ./selinux_share.sh
-```
+   ```
+   cd /path/to/atomic-site/Nulecule/
+   atomicapp genanswers .
```
Comment (TomasTomecek) [inline BUILD.md]: run immediately into this: https://github.com/projectatomic/atomicapp/issues/652

**Comment by TomasTomecek** [comment]
Thanks Josh, works for me now!

(gotta say that volume management with atomicapp is not so easy to figure out at first glance)

**Comment by jberkus** [comment]
Volumes?  Can you explain them to me?  ;-)

**Comment by jberkus** [comment]
Tomas: part of the problem is that the Nulecule spec for Volumes has not been merged: https://github.com/projectatomic/nulecule/pull/203

**Comment by TomasTomecek** [comment]
Thanks for the pointer. Yep, it seems that's it.

**Pull request merged**